contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -3560,6 +3615,22 @@ bbf_ProcessUtility(PlannedStmt *pstmt,
 						else
 							orig_schema = "dbo";
 					}
+					else
+					{
+						owner_oid = rolspec ? get_rolespec_oid(rolspec, true) : InvalidOid;
+						if (OidIsValid(owner_oid) && OidIsValid(db_accessadmin) && !member_can_set_role(GetUserId(), owner_oid) &&
```

Comment:
Both of these owner and db_Accessadmin OIDs must be valid or otherwise we should throw error.

Reply:
Owner oid will only be valid if `rolspec` exists. `rolspec` == NULL means current user should be owner.
for `db_Accessadmin` we remove it based on how we moved forward with [this comment](https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/2970#discussion_r1776827788)

Reply:
Actually I can refactor this a bit.
`else ---> else if (rolspec)` Then owner_oid will always be valid inside the block

Reply:
Done